packages/http/package.json

```diff
@@ -17,10 +17,10 @@
   "main": "lib/index.js",
   "mainSrc": "src/index.ts",
   "scripts": {
+    "build:lib": "talend-scripts build:ts:lib",
```

Comment:
you should add the types entry here to point to lib/index.d.ts
ui-scripts is supposed to warn you about that 🤔

Reply:
<img width="832" alt="CleanShot 2022-02-02 at 09 07 51@2x" src="https://user-images.githubusercontent.com/2909671/152116172-ac890dbc-c45a-4af7-9e46-7aa8f9a246f6.png">

Nothing looks to be warned.
I will add the entry in the package.json

Reply:
it has been added with @talend/scripts-core@11.5.0 and we are at 11.4.0